dataland-frontend/tests/e2e/fixtures/FixtureUtils.ts

```diff
@@ -1,10 +1,12 @@
-import { CompanyReport } from "@clients/backend";
+import { CompanyReport, DocumentReference } from "@clients/backend";
 import { generateCompanyInformation } from "./CompanyFixtures";
 import { getRandomReportingPeriod } from "@e2e/fixtures/common/ReportingPeriodFixtures";
 import { FixtureData } from "@sharedUtils/Fixtures";
 
 export type ReferencedReports = { [key: string]: CompanyReport };
```

Comment:
Do we need both ReferencedReports and ReferencedDocuments?

Reply:
Fixed to be a type that takes both CompanyReports and DocumentReferences as values.

Reply:
ok